stable_baselines3/common/base_class.py

```diff
@@ -874,21 +886,22 @@ def collect_rollouts(self,
                     action_noise.reset()
 
                 # Display training infos
-                if self.verbose >= 1 and log_interval is not None and self._episode_num % log_interval == 0:
+                if log_interval is not None and self._episode_num % log_interval == 0:
+                    stdout = "" if self.verbose >= 1 else "stdout"
```

Comment:
missing comment and maybe add an option in the logger to completely exclude one output, no?
in fact, it would be simpler not to create that output

Reply:
But some logs are still sent to stdout, no?

Reply:
when verbose=0, normally not

Reply:
So no logging at all to stdout when verbose=0?

Reply:
yes ;)